apps/web/app/app/[keyid]/files/file-preview.tsx

```diff
@@ -98,6 +98,9 @@ export default function FilePreview({
                                 <Label htmlFor="filename">Size</Label>
                                 <div className="text-sm">
                                     {(media.size / 1024 / 1024).toFixed(2)} MB
+                                    {/* {media.size >= 1024
```

Comment:
Create a function for this as we also show total size on `settings` screen as well.